SharperUniverse/Core/Builder/ComposeBuilder.cs

```diff
@@ -26,7 +26,8 @@ internal ComposeBuilder(GameRunner game)
         public async Task StartGameAsync()
         {
             ServerLog.LogInfo("Game server now launching...");
-            _game.Server.Start();
+	        _game.PeristenceManager?.Connect();

```

Comment:
`Connect()` should not be on the interface, as this could very well be implementation-specific. Its best to have it as a private piece of logic in the constructor of the persistence manager.

Reply:
Oops, forgot to remove this. Thanks!